spec/requests/evaluators_spec.rb

```diff
@@ -1,37 +1,41 @@
 require 'rails_helper'
 
-RSpec.describe "ManageEvaluators", type: :request do
-  let(:user) { create_and_log_in_user(role: 'challenge_manager') }
+RSpec.describe "Evaluators", type: :request do
+  let(:challenge_manager) { create_and_log_in_user(role: 'challenge_manager') }
   let(:challenge) { create(:challenge) }
   let(:phase) { create(:phase, challenge: challenge) }
   let(:evaluator) { create(:user, role: 'evaluator') }
   let(:invitation) do
     create(:evaluator_invitation, challenge: challenge, phase: phase, email: 'invitation@example.com')
   end
 
-  let(:evaluator_service) { instance_double(EvaluatorManagementService) }
-
-  before do
-    allow(EvaluatorManagementService).to receive(:new).and_return(evaluator_service)
-    ChallengeManager.create(user: user, challenge: challenge)
-    log_in_user(user)
-  end
+  let(:evaluator_service_double) { instance_double(EvaluatorManagementService) }
+  let(:evaluator_service) { EvaluatorManagementService.new(challenge, phase) }
 
   describe 'GET #index' do
     before do
-      invitation # create existing invitation
-      challenge.challenge_phases_evaluators.create(user: evaluator, phase: phase)
+      mock_evaluator_service
+      login_challenge_manager
     end
 
-    it 'assigns @evaluator_invitations and @existing_evaluators' do
-      get phase_manage_evaluators_path(phase)
+    it 'renders invitations and evaluators' do
+      invitation # create existing invitation
+      associate_challenge_manager_challenge
+      associate_evaluator_challenge_phase
+      get phase_evaluators_path(phase)
 
-      expect(assigns(:evaluator_invitations)).to eq([invitation])
-      expect(assigns(:existing_evaluators)).to eq([evaluator])
+      expect(response.body).to include(invitation.email)
+      expect(response.body).to include(evaluator.email)
```

Comment:
expect content in the request response instead of testing assignment of instance variables, which is an implementation detail that could change in the future.